recipes/bactopia/build.sh

```diff
@@ -2,9 +2,8 @@
 mkdir -p ${PREFIX}/bin
 
 chmod 777 bactopia
-chmod 777 bin/setup-datasets.py
-chmod 777 bin/prepare-fofn.py
+chmod 777 bin/helpers/*.py
 
 mv bactopia ${PREFIX}/bin
 mv bin/setup-datasets.py ${PREFIX}/bin/setup-datasets
```

Comment:
Looks like this got renamed.

Reply:
Yep, I was hoping it would get pulled in before the CI check. Should I bump the build number to trigger a new build?

Reply:
I'm at a loss on how to make the updated build.sh recognized in the test build.

Is a new PR required?